blog/migrations/0001_initial.py

```diff
@@ -0,0 +1,66 @@
+# Generated by Django 4.1 on 2024-10-18 19:13
+
+from django.conf import settings
+import django.contrib.auth.models
+import django.contrib.auth.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='User',
```

Comment:
It seems like you are creating a custom User model. Ensure that this is intentional and that you have set AUTH_USER_MODEL in your settings.py to point to this model.